### PR TITLE
CSHARP-2371: Pin Cake.Git to 0.18.0

### PR DESCRIPTION
**Pull request by vincentkam**
https://jira.mongodb.org/browse/CSHARP-2371
Evergreen patch: 
https://evergreen.mongodb.com/version/5b9022b32a60ed15708107d5
Original PR:
https://github.com/mongodb/mongo-csharp-driver/pull/339

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,4 +132,4 @@ Please see our [guidelines](CONTRIBUTING.md) for contributing to the driver.
 * Testo                     test1@doramail.com   
 * Zhmayev Yaroslav          https://github.com/salaros
 
-If you have contributed and we have neglected to add you to this list please contact one of the maintainers to be added to the list (with apologies).
\ No newline at end of file
+If you have contributed and we have neglected to add you to this list please contact one of the maintainers to be added to the list (with apologies).
```
Comment (jyemin) [inline README.md]: Can't see the change on this line.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,7 @@ Please see our [guidelines](CONTRIBUTING.md) for contributing to the driver.
 * Alexander Aramov          https://github.com/alex687
 * Bar Arnon                 https://github.com/I3arnon
 * Wan Bachtiar              https://github.com/sindbach
+* Mark Benvenuto            https://github.com/markbenvenuto
```
Comment (jyemin) [inline README.md:91]: Yay!

**Review by jyemin — APPROVED** [review]
No real expertise to bring to this, but if it works then LGTM.

File: README.md (hunk 1 above)
Comment (vincentkam) [reply README.md]: Thanks for the catch! Newline shenanigans. Perhaps worth standardizing on a "new line at end of file" convention one of these days.

**Comment by vincentkam** [comment]
Merged to master, squashed, commit message tweaked, pushed.

**Pull request closed without merging**